network-proto/node.proto

```diff
@@ -100,6 +105,8 @@ message GossipMessage {
     bytes content = 2;
 }
 
+message Confirmation {}
```

Comment:
In general, it would be nice to declare a separate request and response type for each gRPC call, as these serve as independent extension points for possibly adding fields later.
Streaming payload types are an exception.